app/Migrations/Version20211109143216.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Silex\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+class Version20211109143216 extends AbstractMigration
+{
+    public function up(Schema $schema): void
+    {
+        $schema->getTable('event_recommendations')->addIndex(['event_id']);
```

Comment:
Wasn't it possible to add this line to the migration script that drops the primary key?

Reply:
Unfortunate a drop of a primary key needs one SQL command in one transaction. Also see comment here https://github.com/cultuurnet/udb3-silex/pull/803/commits/23b0ba01ecb09a66440f407c76be39a87e6bff34